apps/web/src/app/[locale]/challenge/[slug]/solutions/_components/solution-editor.tsx

```diff
@@ -117,7 +117,7 @@ export function SolutionEditor({ dismiss, challenge }: Props) {
             Post
           </Button>
         </div>
-        <div className="flex flex-1 flex-col">
+        <div className="relative flex flex-1 flex-col overflow-auto">
```

Comment:
why do we need relative

Reply:
I tried to make the toolbar stick to the top earlier guess I forgot to take that out.